pkgs/by-name/su/superseedr/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "superseedr";
+  version = "0.9.29";
+
+  src = fetchFromGitHub {
+    owner = "Jagalite";
+    repo = "superseedr";
+    rev = "v${version}";
+    hash = "sha256-EszIDtwteskQrD3GBmGsGBnmO+UGE7DSpER7CSSdR2U=";
+  };
+
+  cargoHash = "sha256-HfO+cbrM+WpZDTHB/W92U8EO0yswK4q02SuB3rTZJL4=";
+
+  buildInputs = [
+    openssl
+  ];
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  meta = {
+    description = "BitTorrent Client in your Terminal ";
+    longDescription = ''
+      Superseedr is a modern Rust BitTorrent client featuring a high-performance terminal UI, real-time swarm observability, secure VPN-aware Docker setups, and zero manual network configuration. It is fast, privacy-oriented, and built for both desktop users and homelab/server workflows.
+    '';
+    homepage = "https://github.com/Jagalite/superseedr";
+    license = lib.licenses.gpl3;
```

```suggestion
    license = lib.licenses.gpl3Plus;
```

https://github.com/Jagalite/superseedr/blob/main/Cargo.toml#L10